plugin-flex-ts-template-v2/src/feature-library/conversation-transfer/index.ts

```diff
@@ -17,3 +17,8 @@ export const isColdTransferEnabled = () => {
 export const isMultiParticipantEnabled = () => {
   return enabled && multi_participant;
 };
+
+export const register = (): FeatureDefinition => {
+  if (!enabled) return {};
```

Comment:
Should we change this `enabled` check to match the syntax in the other feature libraries (i.e. creating the `isFeatureEnabled()` function and performing the check with `if (!isFeatureEnabled()) return {};`?

Reply:
Yeah, not sure how this one ended up different.